pkg/configtx/config.go

```diff
@@ -4,7 +4,16 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-// Package configtx allows the creation, retrieval, and modification of channel configtx.
+// Package config provides utilities to create and modify a channel configuration transaction.
+// Channel transactions contain the configuration data defining members and policies for a
+// system chain (ordering service) or channel. A channel transaction can be a transaction that
```

Comment:
`Channel transactions contain the configuration data defining members and policies for a system or application channel and can be used to either create or modify existing channels.`